content/new-author-metadata.yaml

```diff
@@ -39,7 +39,14 @@ author_info:
     affiliations:
       - Innovation Center for Biomedical Informatics, Georgetown University Medical Center, Washington, DC, USA
     funders: NIH R21CA220398, NIH P30CA051008
-  - github: michaelmhoffman
+  - github: alexandari
+    name: Amr Mohamed Alexandari
+    initials: AMA
+    orcid: 0000-0001-8655-8109
+    email: amr.alexandari@gmail.com
+    affiliations:
+      - Department of Computer Science, Stanford University, Stanford, CA
+- github: michaelmhoffman
```

Comment:
Add back the spacing before `- github: michaelmhoffman`

Reply:
This was my merge commit - sorry!